src/components/checkbox/checkbox.api.json

```diff
@@ -42,6 +42,13 @@
     }
   ],
   "snippet": [
-    "<Checkbox value={false$1} onValueChange={() => console.log('value changed')$2}/>"
+    "function Example(props) {",
+    "  const [value, setValue] = useState(false$1);",
+    "  return (",
+    "    <div>",
+    "      <Checkbox value={value$2} onValueChange={setValue$3}/>",
```

Comment:
When selecting the check box there is an issue with the V icon, in the webDemo it works.